htmltruncate.py

```diff
@@ -25,6 +25,8 @@
 
 
 END = -1
+VOID_ELEMENTS = ('area', 'base', 'br', 'col', 'embed', 'hr', 'img', 'input',
```

Comment:
Can you add a link here to https://html.spec.whatwg.org/multipage/syntax.html#void-elements so it's clear to readers why it's correct?